src/TLuaInterpreterMapper.cpp

```diff
@@ -2453,6 +2459,7 @@ int TLuaInterpreter::loadMap(lua_State* L)
         isOk = host.mpConsole->loadMap(location);
     }
     lua_pushboolean(L, isOk);
+    host.mpMap->update();
```

Comment:
Doesn't loading the map automatically draw it anyhow - rendering this redundant?

Reply:
Updated.